lib/pbench/server/auth/__init__.py

```diff
@@ -1,27 +1,21 @@
+import logging
 from http import HTTPStatus
-from typing import Dict, List, Union, Optional
+from typing import Dict, List, Optional, Union
 from urllib.parse import urljoin
 
-import logging
 import jwt
 import requests
```

Comment:
`jwt` and `requests` are out of order

Reply:
Do you mean `import requests` before `import jwt`? but I don't think that's the correct order, is it?